docs/03-github/04-builder.mdx

```diff
@@ -505,6 +517,46 @@ steps:
       sshAgent: ${{ env.SSH_AUTH_SOCK }}
 ```
 
+### Multiple private Github repositories
+
+Github deploy SSH keys are unique per repository. If you have multiple private repositories in your
+packages/manifest.json, you will need to add multiple SSH keys, and use the `webfactory/ssh-agent`
+action to manage them and use the right key for each repository.
+
+In order for `webfactory/ssh-agent` to work inside the container, you will need to upload the public
+keys, ssh config and set the git url mirroring configurations in the container. You can do that by
```

Comment:
Mixed usage of `SSH` and `ssh` throughout the text. I'm on the fence but leaning towards the correct spelling of `SSH Key` with capital letters. And also capital `K` in case of it being used as a keyword, otherwise just `SSH key`.

Reply:
💯